src/components/Modal/Modal.tsx

```diff
@@ -26,10 +26,10 @@ export interface ModalProps extends BaseProps {
   ) => JSX.Element;
 }
 
-export type ModalHandle = {
+export interface ModalHandle {
```

Comment:
Perché è necessario? L'estensione dell'handler della modale usato per useImperativeHandle in che caso potrebbe avvenire? Ho visto che sono stati cambiati anche i type in interface in diverse Props e mi torna, ma questo mi incuriosisce ^^